quelpa.el

```diff
@@ -1633,13 +1632,24 @@ Return t in each case."
 
 (defun quelpa-update-cache (cache-item)
   ;; try removing existing recipes by name
-  (setq quelpa-cache (cl-remove (car cache-item)
-                                quelpa-cache :key #'car))
+  (setq quelpa-cache (--remove (eq (car cache-item) (car it))
```

Comment:
`dash` is a great library, but in general, it would be better to not make changes like this unless they're necessary.  It's easier to review code changes when the diffs only include necessary changes.